shared/js/actions.js

```diff
@@ -145,6 +145,14 @@ loop.shared.actions = (function() {
     }),
 
     /**
+     * Notifies that a tile has been received from the other peer.
+     */
+    ReceivedTile: Action.define("receivedTile", {
+      tile: String
+      // sentTimestamp: String (optional)
```

Comment:
I had intended to switch this to Object as I did in the other place. I'm mildly surprised that even though the types are specified in code they aren't checked.

I wanted to go with Object because I don't think it's clear which fields will be needed here, but I think title and url are a good enough start.

I could see a use for sentTimestamp but I'll remove it.